constants.go

```diff
@@ -669,6 +669,10 @@ const (
 	// during the setup of Connect My Computer. The prefix is followed by the name of the cluster
 	// user. See [teleterm.connectmycomputer.RoleSetup].
 	ConnectMyComputerRoleNamePrefix = "connect-my-computer-"
+
+	// PresetAccessMonitoringRoleName names a preset role that allows
+	// to access and run security reports and audit queries.
+	PresetAccessMonitoringRoleName = "access-monitoring"
```

Comment:
I would have named this "access-monitor" to align with the other preset names. 

For example, we have presets named editor and auditor (not editing and auditing)